plugins/att_cybersecurity_alienvault_otx/plugin.spec.yaml

```
@@ -3,9 +3,9 @@ extension: plugin
 products: [insightconnect]
 name: att_cybersecurity_alienvault_otx
 title: AT&T Cybersecurity AlienVault Open Threat Exchange
-description: This plugin utilizes AT&T Cybersecurity (AlienVault OTX) to retrieve details about an indicator
- Open Threat Exchange is the neighborhood watch of the global intelligence community
-version: 2.0.0
+description: This plugin utilizes AT&T Cybersecurity (AlienVault OTX) to retrieve details about an indicator. Open Threat Exchange is the neighborhood watch of the global intelligence community
+version: 3.0.0
+supported_versions: ["OTX DirectConnect API v1 2022-03-29T14:00:00Z"]
```

The current ISO8601 timestamp is correctly formatted, I simply remove the time component because it is not very valuable information, the date alone is enough and cleaner to read.
```suggestion
supported_versions: ["OTX DirectConnect API v1 2022-03-29"]
```